mautic/mautic
# Contact field mautic tokens in landing pages & page builder autocomplete

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | No
| New feature? | Yes
| Automated tests included? |No
| Related user documentation PR URL | No
| Related developer documentation PR URL | No
| Issues addressed (#s or URLs) | 
| BC breaks? | No
| Deprecations? | No

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Provision to use contact field tokens example {contactfield=firstname} in landing pages.

#### Steps to test this PR:
1. Create or edit any landing page, press ctrl+space in builder and select any contact field token.
2. Check landing page preview, you may see that the contact field tokens are replaced aptly.
